src/coredump/coredumpctl.c

```diff
@@ -236,6 +240,12 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_debugger = optarg;
                         break;
 
+                case 'A':
+                        r = strv_split_full(&arg_debugger_args, optarg, WHITESPACE, EXTRACT_UNQUOTE);
+                        if (r < 0)
+                                return log_error_errno(r, "Failed to parse debugger arguments '%s': %m", optarg);
+                        break;
```

Comment:
this will leak memory if `-A` is used twice on the same cmdline. do this instead:

```c
case 'A': {
        _cleanup_strv_free char **l = NULL;
        r = strv_split_fill(&l, …);
        if (r < 0) …
        strv_free(arg_debugger_args);
        arg_debugger_args = TAKE_PTR(l);
        break;
}
```

i.e. do in new list, and only if that fully worked, free the old and move the new over.